### PR TITLE
Add example of using web routing settings in code

### PR DESCRIPTION
**Pull request by jemayn**
It mentions on https://docs.umbraco.com/umbraco-cms/reference/configuration#reading-configuration-in-code how to read configuration in code, but it only shows the GlobalSettings config which doesn't contain everything.
I've added an example to show how to get options under the WebRoutingSettings.

### CODE REVIEW

File: 11/umbraco-cms/reference/configuration/webroutingsettings.md [hunk 1]
```diff
@@ -26,6 +26,33 @@ An example of a web routing config with default values, and a placeholder for ap
 }
 ```
 
+## Using the webrouting options in code
+
+You can get the config values in code like thise:
```
Comment (sofietoft) [inline 11/umbraco-cms/reference/configuration/webroutingsettings.md:31]: A suggestion to update the wording a bit here, to make a little more clear what's going on.

```suggestion
The following is an example of you can use code to get the value for the `UmbracoApplicationUrl` configuration key:
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @jemayn 💪 

I've made a single suggestion - will also run it by our developers for a quick check.

**Review by nikolajlauridsen — APPROVED** [review]
Looks good to me 👍

File: 11/umbraco-cms/reference/configuration/webroutingsettings.md (hunk 1 above)
Comment (sofietoft) [reply 11/umbraco-cms/reference/configuration/webroutingsettings.md:31]: good catch @jemayn ! 😄 I've updated it now 💪

**Comment by sofietoft** [comment]
Thanks for helping with a review @nikolajlauridsen 🎉 
Looks like everything checks out 💪 
I'll merge the PR.

**Review by sofietoft — APPROVED** [review]

**Pull request merged**